controllers/ovndbcluster_controller.go

```diff
@@ -581,6 +590,76 @@ func (r *OVNDBClusterReconciler) reconcileServices(
 			}
 		}
 	}
+
+	Log.Info("DNS Starts here")
+	if instance.Spec.DBType == v1beta1.SBDBType {
+		Log.Info("Only doing it for SB pods")
+		serviceName = ovndbcluster.ServiceNameSB
+		serviceLabels = map[string]string{
+			common.AppSelector: serviceName,
+		}
+		podList, err = ovndbcluster.OVNDBPods(ctx, instance, helper, serviceLabels)
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		for _, ovnPod := range podList.Items {
+			Log.Info(fmt.Sprintf("Using Pod: %v", ovnPod.Name))
+			var dnsName string
+			var dnsIP string
+			var hostnames []string
+			dnsName = "dns-" + ovnPod.Name
+			// Get Hostname
+			svc, _ := service.GetServiceWithName(
+				ctx,
+				helper,
+				ovnPod.Name,
+				ovnPod.Namespace,
+			)
+			hostname := svc.ObjectMeta.Annotations["dnsmasq.network.openstack.org/hostname"]
```

Comment:
best to use the const from https://github.com/openstack-k8s-operators/infra-operator/blob/main/apis/network/v1beta1/dnsmasq_types.go#L29

Reply:
Oh great! At that time I didn't found it, thanks for the link